build.assets/tooling/cmd/build-apt-repos/config.go

```diff
@@ -0,0 +1,142 @@
+package main
+
+import (
+	"flag"
+	"os"
+	"regexp"
+	"strings"
+
+	"github.com/gravitational/trace"
+	"github.com/sirupsen/logrus"
+)
+
+type Config struct {
+	artifactPath    *string
+	bucketName      *string
+	localBucketPath *string
+	majorVersion    *string
+	releaseChannel  *string
+	logLevel        *uint
+	logJson         *bool
```

Comment:
Make all of these non-pointers. This will mean you won't be able to populate this struct directly with `flag.String` like you do below but this way users of this struct won't have to check each field for nil.

Reply:
Updated pending push